packages/paste-website/src/components/site-wrapper/sidebar/sidebar-disclosure/SidebarDisclosureButton.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import type {BoxProps, BoxStyleProps} from '@twilio-paste/box';
+import {DisclosurePrimitive} from '@twilio-paste/disclosure-primitive';
+import {ChevronDisclosureExpandedIcon} from '@twilio-paste/icons/esm/ChevronDisclosureExpandedIcon';
+
+import {SidebarDisclosureContext} from './SidebarDisclosureContext';
+
+interface SidebarDisclosureButtonProps extends BoxProps {
+  onClick?: () => void;
+  level: 0 | 1;
+}
+
+const StyledButton = React.forwardRef<HTMLButtonElement, SidebarDisclosureButtonProps>(({level, ...props}, ref) => {
+  const levelPaddingMap: Partial<
```

Comment:
NIT: could be moved outside the function, since it's not dependent on the props.